### DISCUSSION TITLE
Single JVM, multiple services. How to identify each service?

### DISCUSSION THREAD

**Discussion by uliss3s** [open-telemetry/opentelemetry-java-instrumentation#10672] [Q&A]
I have two applications running in the same tomcat server instance. How can i identify each service using the java autoinstrumentation agent? If not possible, can this be done by mixing auto and manual instrumentation?

What would be the ideal solution in this case?

**Comment by jaydeluca** [comment]
Something similar came up recently:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/10537 

Is your situation different?
